packages/documentation/src/content/docs/integration/prod/docker-compose.mdx

```diff
@@ -29,34 +29,34 @@ As Let's encrypt certificates are valid for 90 days, you must set up a cron proc
 ```sh
 crontab -e
 ​
-0 3 * * * crontab renew
+0 3 * * * certbot renew
 ```
 
 ## Update DNS records
 
-Next update the DNS records to point to the static external IP address of the volumes:
+Next update the DNS records (A records) to point to the static external IP address of the virtual machine:
 
-| service   | URL              | example                |
-| --------- | ---------------- | ---------------------- |
-| admin     | admin.DOMAIN     | admin.myrafiki.com     |
-| auth      | auth.DOMAIN      | auth.myrafiki.com      |
-| connector | connector.DOMAIN | connector.myrafiki.com |
-| ilp       | ilp.DOMAIN       | ilp.myrafiki.com       |
+| service                       | URL          | example            |
+| ----------------------------- | ------------ | ------------------ |
+| Open Payments resource server | DOMAIN       | myrafiki.com       |
+| ILP Connector                 | ilp.DOMAIN   | ilp.myrafiki.com   |
+| Open Payments auth server     | auth.DOMAIN  | auth.myrafiki.com  |
+| Admin UI                      | admin.DOMAIN | admin.myrafiki.com |
 
 ## Server preparation
 
 Create nginx configuration files for every exposed domain:
```

Comment:
We will need to explain that these files should be in the `/etc/nginx/sites-available` config folder, and then symlinked to the `/etc/nginx/sites-enabled` folder. Maybe would be nice to give the file names to these nginx config files?